module/documents/actor/actor.mjs

```diff
@@ -1105,6 +1133,88 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Initiate concentration on an item.
+   * @param {Item5e} item                     The item on which to being concentration.
+   * @param {object} [effectData]             Effect data to merge into the created effect.
+   * @returns {Promise<ActiveEffect5e[]>}     A promise that resolves to the created and deleted effects.
```

Comment:
```suggestion
   * @returns {Promise<ActiveEffect5e>}     A promise that resolves to the created effect.
```

Reply:
> but as this is an entirely homebrew concept

Not true, as per an example in #3034, a creature from Guildmaster's Guide to Ravnica evidently can conc on two effects.